<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_factory_gameClass</class>
 <widget class="QMainWindow" name="qt_factory_gameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt_factory_game</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QListWidget" name="lbBuildingsList"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Building Data</string>
      </property>
      <widget class="QPlainTextEdit" name="txtBData">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>231</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Global Stats</string>
      </property>
      <widget class="QComboBox" name="cbBuildings">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Housing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Generator</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bar</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="btnBuild">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Build</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelIndex">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>---</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelName">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>---</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_factory_game.qrc"/>
 </resources>
 <connections/>
</ui>
